src/js/game/GameController.js

```diff
@@ -1254,9 +1254,11 @@ class GameController {
       if (blockTypeAtPosition !== "") {
         this.levelModel.destroyBlock(blockIndex);
       }
-      if (this.levelModel.placeBlock(blockType)) {
+
+      const placedBlock = this.levelModel.placeBlock(blockType);
```

Comment:
This is following the pattern in https://github.com/code-dot-org/craft/pull/44/files#diff-bc9f268bfb25698c75c4794fdf5c069fR1347.